src/components/DeleteContactButton.tsx

```diff
@@ -0,0 +1,12 @@
+import handleDeleteButtonClick from "./../utilities";
```

Comment:
nit: Why is the delete button handler in a different file than this one? I would recommend co-locating your code if possible